specs/latest/2.0/index.html

```diff
@@ -4182,6 +4182,19 @@ <h3>GenerateMipmap requires positive image dimensions</h3>
         undefined texture images, skipping an otherwise-common validation.
     </div>
 
+
+    <h3><code>deleteQuery</code> implicitly calls <code>endQuery</code> if the query is active</h3>
+    <p>
+      In GLES, <code>DeleteQueries</code> does not implicitly end queries, even if they are active.
+    </p>
+
+    <div class="note rationale">
+      This behavior was accidental, but was implicitly standardized across browsers by
```

Comment:
Could this be rephrased slightly to say "This behavior was originally implemented to simplify certain WebGL implementations, and was implicitly standardized..."?